internal/cluster_reference_test.go

```diff
@@ -114,7 +114,7 @@ var _ = Describe("ParseRabbitmqClusterReference", func() {
 
 			It("errors", func() {
 				_, _, err := internal.ParseRabbitmqClusterReference(ctx, fakeClient, topology.RabbitmqClusterReference{Name: existingRabbitMQCluster.Name}, existingRabbitMQCluster.Namespace, "")
-				Expect(err).To(MatchError("no status.defaultUser set"))
+				Expect(err).To(MatchError("RabbitmqCluster has no ServiceReference set in status.defaultUser"))
```

Comment:
This can just be `internal.NoServiceReferenceSetError`, so that if we change the content of the error message we need only change in one place.